### PR TITLE
update CMakeLists.txt, fix pkg_check_modules and prevent warning

### PR DESCRIPTION
**Pull request by AlwinEsch**
Also a fix about a typo inside `FindRAW.cmake`, where it define `RAW` to `pkg_check_modules`, but the next want to find `PC_RAW`. This has generated during build:
```
[100%] Linking CXX shared library imagedecoder.raw.so
/usr/bin/ld: -lraw_r kann nicht gefunden werden
collect2: error: ld returned 1 exit status
make[5]: *** [CMakeFiles/imagedecoder.raw.dir/build.make:84: imagedecoder.raw.so.2.0.1] Fehler 1
make[4]: *** [CMakeFiles/Makefile2:73: CMakeFiles/imagedecoder.raw.dir/all] Fehler 2
make[3]: *** [Makefile:152: all] Fehler 2
make[2]: *** [CMakeFiles/imagedecoder.raw.dir/build.make:113: imagedecoder.raw-prefix/src/imagedecoder.raw-stamp/imagedecoder.raw-build] Fehler 2
make[1]: *** [CMakeFiles/Makefile2:138: CMakeFiles/imagedecoder.raw.dir/all] Fehler 2
make: *** [Makefile:84: all] Fehler 2
```

### CODE REVIEW

File: imagedecoder.raw/addon.xml.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="imagedecoder.raw"
-  version="2.0.1"
+  version="2.0.2"
```
Comment (Rechi) [inline imagedecoder.raw/addon.xml.in]: I don't see the need for a version bump.

Comment (AlwinEsch) [reply imagedecoder.raw/addon.xml.in]: I can remove found also another very big bug inside (fix come soon). Like on the other generate it a shared on depends, where is after zip created no more available and the addon can never work by the repo way!

File: depends/common/libraw/CMakeLists.txt [hunk 2]
```diff
@@ -7,7 +7,12 @@ include(ExternalProject)
 externalproject_add(libraw
   SOURCE_DIR ${CMAKE_SOURCE_DIR}
   UPDATE_COMMAND autoreconf -vif
-  CONFIGURE_COMMAND CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR} ${CMAKE_SOURCE_DIR}/configure --disable-examples --prefix=${OUTPUT_DIR}
+  CONFIGURE_COMMAND CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR}
+                    ${CMAKE_SOURCE_DIR}/configure
+                      --disable-examples
+                      --disable-shared
+                      --with-pic
+                      --prefix=${OUTPUT_DIR}
```
Comment (AlwinEsch) [inline depends/common/libraw/CMakeLists.txt:18]: Also by this without version change? This is the major fix.

Comment (Rechi) [reply depends/common/libraw/CMakeLists.txt:18]: I'll handle creating a new release after this PR is merged.

**Comment by AlwinEsch** [comment]
Also here worked travis now.

**Comment by AlwinEsch** [comment]
@Rechi can you describe the:
```
continuous-integration/jenkins/branch Expected — Waiting for status to be reported
```
This goes not away?

**Pull request merged**